### PR TITLE
Trilinos: fix inconsistencies.

### PR DESCRIPTION
**Pull request by dpo**
Some dependencies are currently commented out.
Comment out corresponding code.

### CODE REVIEW

File: trilinos.rb [hunk 1]
```diff
@@ -216,19 +216,19 @@ def install
 
     args << onoff("-DTPL_ENABLE_SCALAPACK:BOOL=",   (build.with? "scalapack"))
 
-    args << onoff("-DTPL_ENABLE_SuperLU:BOOL=",     (build.with? "superlu"))
-    args << "-DSuperLU_INCLUDE_DIRS=#{Formula["superlu"].opt_include}/superlu" if build.with? "superlu"
+    args << onoff("-DTPL_ENABLE_SuperLU:BOOL=", "OFF") #   (build.with? "superlu"))
```
Comment (davydden) [inline trilinos.rb]: shouldn't here (and elsewhere) be `0`/`false` instead of `OFF`?

**Comment by dpo** [comment]
We use `ON`/`OFF` everywhere: https://github.com/Homebrew/homebrew-science/pull/2759/files#diff-06990b9d06d23ea1ef3926d6f3944c26R77
Has anything changed in that respect?

File: trilinos.rb (hunk 1 above)
Comment (dpo) [inline trilinos.rb]: Sorry, I misunderstood your comment. Yes, it should be `false`. Thanks.

**Comment by davydden** [comment]
@BrewTestBot test this please

**Comment by davydden** [comment]
failures in `brew doctor`, otherwise pass.

**Pull request closed without merging**